src/Components/ShowAllArticles.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function ShowAllArticles({ selectedArticles }) {
+  const [allArticles, setAllArticles] = useState([]);
+  const [IsLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(`https://nc-news-nikhil.herokuapp.com/api/articles`)
+      .then((res) => {
+        return res.json();
+      })
+      .then(({ articles }) => {
+        console.log(articles);
+        setAllArticles(articles);
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (IsLoading) {
+    return <p>Loading......</p>;
+  }
+  return (
+    <>
+      <h4>Show All articles: </h4>
+
+      <div className="ShowAllArticles">
+        {allArticles.map((article) => {
+          return (
+            <div key={article.article_id} className="ShowAllArticles__Article">
+              <h4 className="thumbnail">
```

Comment:
Could these be their own card component? Would look a bit nicer. 

Some responsiveness for when it gets bigger - cards next to next other, instead of stacked.

Reply:
Maybe with abit more detail about each other them, ie. author, date created etc.